src/components/History/History.tsx

```diff
@@ -62,13 +63,28 @@ const History: React.FC<HistoryProps> = ({
     return sortArrayByDate(history)
   }, [history])
 
+  useEffect(() => {
+    trackEvent('hl_chat_history_visibility_changed', { 
+      isVisible: showHistory,
+      historyItemCount: history.length
+    });
```

Comment:
Can likely remove this and just use the toggle event.

Reply:
Removed